packages/driver/src/cy/commands/prompt/prompt-driver-types.ts

```diff
@@ -0,0 +1,7 @@
+export interface CypressInternal extends Cypress.Cypress {
+  backend: (eventName: string, ...args: any[]) => Promise<any>
```

Comment:
seems like this would be defined somewhere else?

Reply:
The typing of this is a little weird. We only expose `Cypress.backend('net')` publicly from what i can tell.

I think having a direct `Cypress.promptBackend` will make some of this easier.

Reply:
Yeah, for sure, however, I do see `backend` defined internally [here](https://github.com/cypress-io/cypress/blob/2e0a738678fcd751887c4e92cbaeedaea34322ea/packages/driver/types/internal-types.d.ts#L42).

Reply:
Right. But these are the types that are "delivered" from the cloud which does not have access to the Cypress internals. Kind of similar to what had to be done here: https://github.com/cypress-io/cypress-services/blob/develop/app/packages/studio/src/app/types.ts#L18-L21

Reply:
Added a comment to clarify: [`dcd49e8` (#31748)](https://github.com/cypress-io/cypress/pull/31748/commits/dcd49e89dfce64cb564d1374fbcf7bf75acab391)